src/core/loaders/Loader.ts

```diff
@@ -11,6 +11,8 @@ export abstract class Loader extends Disposable {
   protected _onDidChange = new EventEmitter<string>()
   readonly onDidChange = this._onDidChange.event
 
+  readonly nestedPluralizationKeys = ['one', 'other', 'zero', 'two', 'few', 'many']
```

Comment:
Nit on convention; should this be a const named in screaming snake case, maybe even defined outside & above this class?

```suggestion
  const NESTED_PLURALIZATION_KEYS = ['one', 'other', 'zero', 'two', 'few', 'many']
```